covid_xprize/examples/prescriptors/neat/utils.py

```diff
@@ -110,14 +122,10 @@ def get_predictions(start_date_str, end_date_str, pres_df, countries=None):
     ips_df = pd.concat([hist_df, pres_df])
 
     # Write ips_df to file
-    ips_df.to_csv(TMP_PRESCRIPTION_FILE)
+    ips_df.to_csv(TMP_PRESCRIPTION_FILE_NAME)
 
     # Use full path of the local file passed as ip_file
-    ip_file_full_path = os.path.abspath(TMP_PRESCRIPTION_FILE)
-
-    # Go to covid-xprize root dir to access predict script
-    wd = os.getcwd()
```

Comment:
No longer any need to jump around between directories as we locate things relative to where the script itself lives.